lib/pluginmanager/util.rb

```diff
@@ -17,6 +19,7 @@ def self.logstash_plugin?(plugin, version = nil)
       end
     else
       dep = Gem::Dependency.new(plugin, version || Gem::Requirement.default)
+      Gem.sources = Gem::SourceList.from(options[:sources]) if options[:sources]
```

Comment:
Hi,
  I used sources here because using rubygems_sources is confusing in the context of other gem repositories. I will change this here to be in the same page as bundler, as you requested.